packages/wp-now/src/run-cli.ts

```diff
@@ -42,6 +42,11 @@ export async function runCli() {
 					type: 'string',
 					default: DEFAULT_PHP_VERSION,
 				});
+				yargs.option('wp', {
+					describe: "WordPress version to use: e.g. '--wp=6.2'",
+					type: 'string',
+					default: DEFAULT_WORDPRESS_VERSION,
```

Comment:
If I don't include `--wp=<version>` when I run `wp-now start`, I see this error:

```
Starting the server......
Project directory: /Users/danielbachhuber/projects/gutenberg
mode: plugin
php: 8.0
wp: latest
WordPress latest folder already exists. Skipping download.
TypeError: Cannot destructure property 'downloaded' of '(intermediate value)' as it is undefined.
    at downloadWordPress (file:///Users/danielbachhuber/projects/wordpress-playground/dist/packages/wp-now/main.js:122:11)
    at async _WPNow.start (file:///Users/danielbachhuber/projects/wordpress-playground/dist/packages/wp-now/main.js:365:5)
    at async startServer (file:///Users/danielbachhuber/projects/wordpress-playground/dist/packages/wp-now/main.js:500:3)
    at async Object.handler (file:///Users/danielbachhuber/projects/wordpress-playground/dist/packages/wp-now/main.js:605:9)
Failed to start the server: Cannot destructure property 'downloaded' of '(intermediate value)' as it is undefined
```

Reply:
Fixed! https://github.com/WordPress/wordpress-playground/pull/252/commits/4ca05f6d451284be00a616d49f7aed55b34e2638